skrobot/viewers/_trimesh.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, resolution=None):
         self._links = collections.OrderedDict()
 
         self._redraw = True
-        pyglet.clock.schedule_interval(self.on_update, 1 / 30)
+        pyglet.clock.schedule_interval(self.on_update, 1.0)
```

Comment:
Thanks!
How about making this a variable and making it an argument of the `TrimeshSceneViewer` class?

Reply:
Thank you for your review!

I updated in https://github.com/iory/scikit-robot/pull/239/commits/a75374c95bf9ab6452d5fde86fe41cedc63cf5d3